drools-completion/pom.xml

```diff
@@ -15,6 +15,18 @@
 
   <dependencies>
 
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-compiler</artifactId>
+      <version>7.66.0.Final</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-mvel</artifactId>
+      <version>7.66.0.Final</version>
```

Comment:
@lbenc135 any feedback on this? ^

Reply:
Sorry for the delay. This is a heavy work-in-progress and I was just trying to get something working. I didn't worry about the dependencies or versions yet.

Reply:
@lbenc135 If you're saying that this pull request is a draft and you're still working on it I will wait till it will be ready before reviewing and merging it. Conversely, if you want this pull request to be merged as it is despite being still a work in progress, that's also ok for me, but please fix this problem with dependencies first: for my experience bringing in unwanted/unnecessary dependencies immediately causes a technical debt from which it's much harder to recover at a later stage. Thanks.

Reply:
It's definitely a draft, and nowhere near ready to be merged. Unfortunately I probably won't have any time to finish it at least for a month, so if there's anyone willing to take it over that would be great. Also, there's still a lot of open questions about how to implement this feature correctly.